app/src/main/java/com/cornellappdev/uplift/ui/MainNavigationWrapper.kt

```diff
@@ -140,7 +148,7 @@ fun MainNavigationWrapper(
     ) {
         NavHost(
             navController = navController,
-            startDestination = "home",
+            startDestination = "onboarding",
             modifier = Modifier.padding(it)
         ) {
```

Comment:
We should have a follow-up PR pretty soon about saving to local storage whether or not the user has already completed onboarding or not. It should be sufficient to just save this to local preferences and read it upon the app opening, changing the start destination accordingly.